Assets/HoloToolkit/Input/Scripts/Gaze/GazeManager.cs

```diff
@@ -64,9 +64,9 @@ public class GazeManager : Singleton<GazeManager>, IPointingSource
         /// Origin of the gaze.
         /// </summary>
         public Vector3 GazeOrigin
-        {
-            get { return Ray.origin; }
-        }
+        { get; protected set; }
+     // return Ray.origin; 
```

Comment:
we can probably remove this commented out part.

Do we handle setting this later?

Reply:
Yeah, sorry for the slop there.

Reply:
Was in the habit of just commenting out the original HoloToolkit code as a reference.